go/libkb/sig_chain.go

```diff
@@ -343,6 +348,12 @@ func (sc *SigChain) GetCurrentSubchain(eldest keybase1.KID) (links []*ChainLink,
 	l := len(sc.chainLinks)
 	lastGood := l
 	for i := l - 1; i >= 0; i-- {
+
+		if sc.chainLinks[i].IsStubbed() {
+			lastGood = i
+			continue
+		}
```

Comment:
This feels dangerous to me. Can the server commit Heinous Mischief by stubbing out an eldest-type link or something like that?